src/components/core/pivotTable/agGridApiWrapper.ts

```diff
@@ -0,0 +1,13 @@
+// (C) 2007-2019 GoodData Corporation
+function getHeaderHeight(gridApi: any): number {
```

Comment:
`gridApi` doesn't have interface?

Reply:
I've put `GridApi` interface as a parameter type, but now we need to retype it when using within function (which is still better, because we check we have really received `GridApi` object as a parameter).